CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ This is a non-mandatory, but highly recommended, release targeting the mainnet n
 - [#5512](https://github.com/ChainSafe/forest/pull/5512) Fixed `Filecoin.EthTraceFilter` RPC method.
 - [#5517](https://github.com/ChainSafe/forest/issues/5517) Fix the `forest-cli sync wait` issue
 - [#5540](https://github.com/ChainSafe/forest/pull/5540) Avoid duplicate migrations.
+- [#5559](https://github.com/ChainSafe/forest/pull/5559) Change `Filecoin.ChainGetMinBaseFee` to `Forest.ChainGetMinBaseFee` with read access
```

Comment:
This release was already done. This entry should be below `unreleased` in the `breaking` section.

Reply:
Updated.